hera_mc/part_connect.py

```diff
@@ -383,9 +383,10 @@ def connection(self, **kwargs):
             setattr(self, key, value)
 
 
-def stop_existing_connections(session, h, conn_list, at_date, actually_do_it):
+def stop_existing_connections_to_part(session, h, conn_list, at_date, actually_do_it):
     """
-    This adds stop times to the previous connections in conn_list
+    This adds stop times to the previous connections in conn_list.  This is probably 
+    a dangerous module as is.
```

Comment:
Why is it dangerous? And where is it being called? I don't see it being used anywhere else in the PR.